content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -84,17 +84,15 @@ matching requests should flow through. If all requests continue to be denied, yo
 
 1. Make sure there is no typo in your policy YAML file.
 
-1. Avoid enabling authorization for Istio Control Planes Components, including Mixer, Pilot, Ingress. Istio authorization policy is designed for authorizing access to services in Istio Mesh. Enabling it for Istio Control Planes Components may cause unexpected behavior.
+1. Avoid enabling authorization for istiod. Istio authorization policy is designed for authorizing access to workloads in Istio Mesh. Enabling it for istiod may cause unexpected behavior.
 
-1. Make sure that your `ServiceRoleBinding` and referred `ServiceRole` objects are in the same namespace (by checking "metadata"/”namespace” line).
+1. Make sure that your authorization policies are in the right namespace (by checking "metadata"/”namespace” line).
 
-1. Make sure that your service role and service role binding policies don't use any HTTP only fields
-for TCP services. Otherwise, Istio ignores the policies as if they didn't exist.
+1. Make sure that your authorization policies with ALLOW action don't use any HTTP only fields for TCP traffic.
+Otherwise, Istio ignores the ALLOW policies as if they don't exist.
 
-1. In Kubernetes environment, make sure all services in a `ServiceRole` object are in the same namespace as the
-`ServiceRole` itself. For example, if a service in a `ServiceRole` object is `a.default.svc.cluster.local`, the `ServiceRole` must be in the
-`default` namespace (`metadata/namespace` line should be `default`). For non-Kubernetes environments, all `ServiceRoles` and `ServiceRoleBindings`
-for a mesh should be in the same namespace.
+1. Make sure that your authorization policies with DENY action don't use any HTTP only fields for TCP traffic.
```

Comment:
This and the above entry seem to be a convoluted way of saying that the HTTP only fields are ignored, which is what it already says in https://deploy-preview-6767--preliminary-istio.netlify.com/docs/concepts/security/#using-istio-authorization-on-plain-tcp-protocols. I would suggest just one entry that says something like:
```
Make sure that authorization policies for TCP workloads don't use any HTTP only fields.
HTTP only fields are ignored for TCP traffic and consequently have no effect on the policy.
```